nemo_curator/classifiers/quality.py

```diff
@@ -12,22 +12,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 import os
-
-os.environ["RAPIDS_NO_INITIALIZE"] = "1"
 from dataclasses import dataclass
 
-import torch
-import torch.nn as nn
+os.environ["RAPIDS_NO_INITIALIZE"] = "1"
 from crossfit.backend.torch.hf.model import HFModel
-from transformers import AutoConfig, AutoModel
-from transformers.models.deberta_v2 import DebertaV2TokenizerFast
+from transformers import AutoConfig, AutoTokenizer
 
 from nemo_curator.classifiers.base import (
     DistributedDataClassifier,
+    HFCustomModel,
     _run_classifier_helper,
 )
 from nemo_curator.datasets import DocumentDataset
 
+QUALITY_IDENTIFIER = "nvidia/quality-classifier-deberta"
+
 
 @dataclass
 class QualityModelConfig:
```

Comment:
There's a lot of repeated/identical logic for `DomainModelConfig` vs `QualityModelConfig`, `DomainModel` vs `QualityModel`, and `DomainClassifier` vs `QualityClassifier`. I was wondering how much of this I should try to consolidate versus keeping them separate?

Reply:
I think you've done a good job so far. We'll keep it with what you have for now.